### ISSUE TITLE
Openelec 4.0.6 Update fail 

### ISSUE THREAD

**Issue by klinnex** [maxnet/berryboot#126]
Hello,

OpenELEC 4.0.6 on berryboot don't update.

When the file are in update folder they don't take effet on reboot.

How can i solve that.

**Comment by klinnex** [comment]
I have check the share "Update", the file of the update exist.

They seem on reboot, they don't check it.

**Comment by DcR-NL** [comment]
Same issue here. I tried updating from the menu to 4.1.2. It started downloading, extracting and after a reboot it still was version 4.0.6. 

Then I tried to download the file myself, placing it in the update folder and rebooting (OpenELEC-RPi.arm-4.2.1.tar and OpenELEC-RPi.arm-5.0.0.tar). Even tried extracting the tars (which shouldn't be necessary anymore from v4.0.0+). But nothing, still v4.0.6... Very frustrating.

//Edit:
Moved to NOOBS Lite v1.3.10. Comes with OpenELEC v4.0.5. But managed to update it to v5.0.0 by placing the OpenELEC-RPi.arm-5.0.0.tar file in the Update directory (through Samba) and rebooting. :)

Did this project die because of NOOBS's multi-boot feature?
